src/EmployeeBook.java

```diff
@@ -0,0 +1,240 @@
+public class EmployeeBook {
+    private Employee[] workers;
+
+    public void printWorker() {
+        for (int i = 0; i < workers.length; i++) {
+            if (workers[i] != null) {
+                System.out.println(workers[i].toString());
+            }
+        }
+    }
+
+    public EmployeeBook(int len) {
+        if (len < 0) {
+            throw new IllegalArgumentException("Длина не может быть отрицательной");
+        }
+        this.workers = new Employee[len];
+    }
+
+
+    public float summarize() {
+        float count = 0;
+        for (int i = 0; i < workers.length; i++) {
+            if (workers[i] != null) {
+                count += workers[i].getSalary();
+            }
+        }
+        return count;
+    }
+
+    public Employee getMin() {
+        Employee name = null;
+        float count = 1000000000f;
```

Comment:
а если у кого-то будет такая ЗП?) Лучше здесь просто взять максимум для этого типа
```suggestion
        float count = Float.MAX_VALUE;
```